apm-sniffer/apm-sdk-plugin/graphql-plugin/graphql-12.x-15.x-plugin/src/main/resources/skywalking-plugin.def

```diff
@@ -14,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-graphql=org.apache.skywalking.apm.plugin.graphql.v12.define.GraphqlInstrumentation
+graphql=org.apache.skywalking.apm.plugin.graphql.v12tov15.define.GraphqlInstrumentation
```

Comment:
Should we change the key?

Reply:
Like Vertx plugin, we have `vertx-core-3.x` for version 3.x. It is better we could have graphql version in the key. Notice, once you change, the Plugin-list.md should be changed too.